src/Illuminate/Foundation/Console/KeyGenerateCommand.php

```diff
@@ -33,12 +33,18 @@ class KeyGenerateCommand extends Command
      *
      * @return void
      */
-    public function handle()
+    public function handle(): void
```

Comment:
```suggestion
    public function handle()
```

Let's avoid changing signature on existing `public` and `protected` method in a minor/patch release as this command may be extended by package or application.